Products/PluggableAuthService/events.py

```diff
@@ -40,6 +45,22 @@ def __init__(self, principal):
         self.object = principal
 
 
+@implementer(IPrincipalAddedToGroupEvent)
+class PrincipalAddedToGroup(PASEvent):
+
+    def __init__(self, principal, group_id):
```

Comment:
Either this should be `principal_id` or below `principal_id` should be replaced by `principal`.

Reply:
As I wrote in the description, I have elected to keep the name `principal` because that's what the base class uses.

Reply:
(there's no `principal_id below`, not sure what the issue is)

Reply:
I approve the PR because I do not want to block this one, but it seems to me that if we use principal here we should also use it everywhere, so also below where the code does:
```
notify(PrincipalRemovedFromGroup(principal_id, group_id))
```
When I read `principal` I think to an object, when I read `principal_id` I think to a string :)

Being coherent will decrease confusion.

Reply:
Ah, I see what you mean. In ZODBGroupManager.py `principal_id` just happens to be what's available in the mmethod that does the notification.